impf/constructors.py

```diff
@@ -20,6 +20,7 @@ def browser_options():
     if os.environ.get('DOCKER_ENV'):
         opts.add_argument('--no-sandbox')
         opts.add_argument('--disable-dev-shm-usage')
+    opts.add_experimental_option("excludeSwitches", ["enable-logging"])
```

Comment:
Good find! Seems odd it's a `list` instead of a `str`; but then again it's Google Chrome, so I wouldn't be surprised.

Reply:
Solution is from here: [https://stackoverflow.com/questions/64634041/is-there-a-method-or-a-function-to-hide-seleniums-output-from-the-program](https://stackoverflow.com/questions/64634041/is-there-a-method-or-a-function-to-hide-seleniums-output-from-the-program)